### ISSUE TITLE
DataAnnotations not effective

### ISSUE THREAD

**Issue by elitesw** [abpframework/abp#15410]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Description

In blazor wasm mode, data annotations not effective.  The culture has been set to zh-Hans. Call the API from Swagger and the return is correct.

![test01](https://user-images.githubusercontent.com/54815375/212495778-f53ffc41-a6b2-4496-92f2-c7ee8b934fce.png)

![test02](https://user-images.githubusercontent.com/54815375/212496374-1aa1fa23-e1a8-4077-b0fd-1cd58c8f0347.png)

### Version

6.02

### User Interface

Blazor

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

Tiered

### Operation System

Windows (Default)

**Comment by realLiangshiwei** [comment]
It looks like you are using a third-party UI, please open an issue on the relevant repo.

**Comment by elitesw** [comment]
Thank you for your reply! 
It should not be the reason for the third-party UI. I tried to use the built-in EditForm and ValidationSummary tag, and the display is the same. The validation error text is the content defined by the AbpValidationResource. I found that there is an AbpDataAnnotationsLocalizationOptions configuration option in MVC mode. There is no similar in Blazor.
![test03](https://user-images.githubusercontent.com/54815375/212734755-e5b37898-357c-4c46-8beb-7ee131a85c10.png)
